akka-remote/src/main/scala/akka/remote/serialization/SystemMessageSerializer.scala

```diff
@@ -0,0 +1,161 @@
+package akka.remote.serialization
```

Comment:
I'm reviewing what I missed over the last days, so forgive my boring comment here: missing license ;)